src/api/test/functional/webui/user_controller_test.rb

```diff
@@ -181,6 +181,22 @@ def test_that_redirect_after_login_works # spec/controllers/webui/users_spec.rb
     assert_equal search_path, current_path
   end
 
+  def test_that_redirect_from_user_do_login_works # spec/controllers/webui/users_spec.rb
```

Comment:
Please remove the comment. Those are just used by us to mark tests we already moved to rspec. 

See https://github.com/openSUSE/open-build-service/tree/master/src/api/spec for details